Milliner/cfg/config.example.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+fedora_base_url: http://localhost:8080/fcrepo/rest
+drupal_base_url: http://localhost:8000
+
+db.options:
+  driver: pdo_mysql
+  host: 127.0.0.1
+  port: 3306
+  dbname: gemini
+  user: changeme
+  password: changeme
```

Comment:
Oh I guess it does, so Gemini and Milliner both need a PathMapper DB connection.